### ISSUE TITLE
Is full support of 1.10 still planned?

### ISSUE THREAD

**Issue by GeorgeLubaretsi** [django-salesforce/django-salesforce#173]
Readme says Q4 of 2016. Any plans on development? Thanks!

**Comment by philchristensen** [comment]
Well, to be honest, the level of support hasn't changed substantially since 1.8. Basic to intermediate ORM support is still totally functional. IIRC, it's mostly just raw() and values() that aren't supported.

Is there a particular feature (or lack thereof) that you're blocked by?

**Comment by jsma** [comment]
The comments in #160 make it sound like django-salesforce is not currently compatible with Django 1.10. @hynekcer's comments suggest he was aiming for a 0.7 release that would be Django 1.10 compatible. Is Django 1.10 already supported in 0.6.9? I had held off on upgrading to Django 1.10 because I thought #160 was a known blocker.

**Comment by hynekcer** [comment]
The support for Django 1.10 is ready for a long time (about half year) in my repository and I use it regularly.
`https://github.com/hynekcer/django-salesforce/tree/fix167` I install it as editable:

    pip install -e git+https://github.com/hynekcer/django-salesforce.git@b92c5f2c2c97b3434d554fd5a90aaa5a4ce9134b#egg=django-salesforce

It is not a high quality code as used by me. I like to reorganize the code so that everything works after every commit and every commit can be understand, but this is not the case. Some functionality is duplicated in a new and old code, but the old code can not be still removed. Something new is still not clear, that it will be useful. Some tests are failing now, but I think it is because I started to test it more exactly. I asked people to try this code, but without feedback is no progress.

**Comment by philchristensen** [comment]
Thanks @hynekcer, i was just about to print something similar. This remains an independent, unfunded, open-source project, so my position is that we'll get to Django 1.10 ASAP, but that may be awhile. The best way to contribute is to test the branches @hynekcer has indicated, which will help us get a formal release out for 1.10

**Comment by philchristensen** [comment]
I just pushed a release for v0.7.0 including updated support for 1.10 and 1.11.  Is this ticket okay to be closed?

https://pypi.python.org/pypi/django-salesforce/0.7.0

**Comment by hynekcer** [comment]
No, the solution depends on the new code from August in the development repository that tries to implement some parts of standard DB API, but it is a "double-track" solution. There are some unused and untested  parts. See also the [comment](https://github.com/django-salesforce/django-salesforce/pull/183#discussion_r114441352)
written yesterday.

The name of this issue looks terrible after many months after our promise. I simplify probably the new code temporarely without ambition to say that I'm implementing the DB API still now.
